duration max_rtt
duration max_duration_throughput
---
Header header
duration rtt
float64 throughput      # Mbps